.github/Dockerfile.base

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:22.04
+SHELL ["/bin/bash", "-c"]
+
+# Set environment variables
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
```

Comment:
Can we split dependencies by categories? E.g. docs, ffe, etc. Just to manage them easier for future changes :))